.github/workflows/pr_perf_levm.yaml

```
@@ -77,78 +48,114 @@ jobs:
           ref: main
 
       - name: Install Rust
-        uses: dtolnay/rust-toolchain@1.81.0
+        uses: dtolnay/rust-toolchain@1.87.0
         with:
           components: rustfmt, clippy
 
       - name: Set up cargo cache
         uses: Swatinem/rust-cache@v2
 
-      - name: Install hyperfine
-        uses: taiki-e/install-action@v2
-        with:
-          tool: hyperfine@1.16
-
-      - name: Install solc
-        uses: pontem-network/get-solc@master
-        with:
-          version: v0.8.29
-
-      - name: Run main benchmarks
+      - name: Build main benchmark
         run: |
           cd crates/vm/levm
-          make revm-comparison-ci
-          {
-            echo "#### Benchmark Results: Factorial";
-            cat Factorial.md || echo "No results";
-            echo "#### Benchmark Results: Factorial - Recursive";
-            cat FactorialRecursive.md || echo "No results";
-            echo "#### Benchmark Results: Fibonacci";
-            cat Fibonacci.md || echo "No results";
-            echo "#### Benchmark Results: ManyHashes";
-            cat ManyHashes.md || echo "No results";
-            echo "#### Benchmark Results: BubbleSort";
-            cat BubbleSort.md || echo "No results";
-            echo "#### Benchmark Results: ERC20 - Transfer";
-            cat ERC20Transfer.md || echo "No results";
-            echo "#### Benchmark Results: ERC20 - Mint";
-            cat ERC20Mint.md || echo "No results";
-            echo "#### Benchmark Results: ERC20 - Approval";
-            cat ERC20Approval.md || echo "No results";
-          } > main_result.md
-
-      - name: Upload main results
+          make build-revm-comparison
+      - name: Upload main benchmark binary
         uses: actions/upload-artifact@v4
         with:
           name: main-result
```

```suggestion
          name: main-binary
```
For both `pr-result` and `main-result` that are here it seems they don't represent the markdown result anymore. They represent the benchmark binary, so I consider appropriate changing these names.